canister/src/api/fee_percentiles.rs

```diff
@@ -392,42 +392,6 @@ mod test {
         });
     }
 
-    #[test]
-    fn get_current_fee_percentiles_big_input() {
-        let number_of_blocks = 1_000;
-        let initial_balance = 500_500; // number_of_blocks * (number_of_blocks + 1) / 2
-        let blocks = generate_blocks(initial_balance, number_of_blocks);
-        let stability_threshold = blocks.len() as u128;
-        init_state(blocks, stability_threshold);
```

Comment:
I deleted this test because with block header validation its runtime jumped from ~2 seconds to about 3 minutes. The reason it's slow is because it's using a stability threshold of > 500k, whereas we don't intend to have a stability threshold that's higher than 144 at this time. The code path can be optimized, but given that it's not something we intend to support, and given that this test (AFAICT) doesn't add extra coverage, I think deleting it would be the simplest. @maksymar WDYT?